packages/react-components/src/types/ParticipantListParticipant.ts

```diff
@@ -22,6 +22,19 @@ export type CallParticipantListParticipant = ParticipantListParticipant & {
   /* @conditional-compile-remove(rooms) */
   /** Role of participant in Rooms call */
   role?: Role;
+  /* @conditional-compile-remove(raise-hand) */
+  /** Whether calling participant is raised hand */
+  raisedHand?: RaisedHand;
+};
+
+/* @conditional-compile-remove(raise-hand) */
+/**
+ * Raised hand state with order
+ *
+ * @public
+ */
+export type RaisedHand = {
+  order: number;

```

Comment:
nit: maybe call this something like `raisedHandOrderPosition`

Reply:
changed to 20